boot/bootutil/include/bootutil/bootutil.h

```diff
@@ -26,13 +26,15 @@
 extern "C" {
 #endif
 
-/** Attempt to boot the contents of slot 0. */
+/** Attempt to boot the contents of primary slot. */
```

Comment:
I'll just put one of these here, but there is lots below. "slot 0" and "primary slot" function differently in English, which makes a majority of the uses below ungrammatical. The phrase "slot 0" is definite, and does not use an article. However, "primary slot" does require an article. "Attempt to boot the contents of the primary slot". I would suggest considering terminology that doesn't invoke a grammatical change, or if made, we have a lot more things to change in this commit.

Reply:
Thank you @d3zd3z for the feedback. I will fix these grammatical faults, however before it I would like to reach a consensus regarding the terminology change.